recipe/meta.yaml

```diff
@@ -1,31 +1,40 @@
-{%set name = "sklearn-contrib-lightning" %}
-{%set version = "0.4.0" %}
-{%set hash_type = "md5" %}
-{%set hash_val = "7324683380bb8a5d9257817bd549728d" %}
+{% set name = "sklearn-contrib-lightning" %}
+{% set version = "0.5.0" %}
+{% set hash_type = "md5" %}
+{% set hash_val = "dca6cf1bea09cf77cf052119f4a43093" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/scikit-learn-contrib/lightning/archive/{{ version }}.tar.gz
```

Comment:
Why this change?

Reply:
The version on pypi does not include the Makefile. Running `make clean` is necessary as the tarball includes some cython-generated files that break the build.